index.js

```diff
@@ -10,32 +10,34 @@ module.exports = function(homebridge) {
 
 function HomebridgeRoutineTriggerAccessory(log, config) {
   var accessory = this;
```

Comment:
If you are replacing all references to the `accessory` var, then it may as well be removed